extensions/renderer/resources/app_window_custom_bindings.js

```diff
@@ -30,9 +30,10 @@ var try_hidden = function (view) {
 };
 
 var try_nw = function (view) {
-  if (view.nw)
-    return view;
-  return privates(view);
+  if (view.nw) {
+    return true;
+  }
+  return false;
 };
```

Comment:
@xzhan96 as I said, other code expects try_nw() returning a object, but your patch changes it to return a boolean, that's obviously a breaking change. Please keep it untouched and fix the getAll() function().